src/app/component/CartCoupon/CartCoupon.style.scss

```diff
@@ -0,0 +1,36 @@
+.CartCoupon {
+    @include mobile {
+        width: 100%;
+    }
+
+    &-Message {
+        display: inline-block;
+    }
+
+    &-Input {
+        display: inline-block;
+
+        input {
```

Comment:
Please check if it is possible to set width to input, without styling the `input` itself.

Reply:
Checked multiple examples where <Field type="text'> is used. Everywhere width is set directly to `input`.

Reply:
Resolved